packages/wonder-blocks-core/components/clickable-behavior.test.js

```diff
@@ -106,8 +106,8 @@ describe("ClickableBehavior", () => {
         );
         expect(button.state("focused")).toEqual(false);
         button.simulate("keydown", {keyCode: keyCodes.space});
-        button.simulate("click", {preventDefault: jest.fn()});
         button.simulate("keyup", {keyCode: keyCodes.space});
+        button.simulate("click", {preventDefault: jest.fn()});
```

Comment:
This is the correct order of events. (Doesn't change the result of the test for neither the existing version nor my change though.)